content/docs/esc/development/psp.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Pulumi Service Provider
+title_tag: Manage Pulumi ESC resources using a Pulumi Program
+h1: Manage Pulumi ESC resources using a Pulumi Program
+meta_desc: Pulumi Service Provider allows you to create and manage ESC resources like environments, permissions and revisions tags using a Pulumi program.
+menu:
+  esc:
+    identifier: esc-psp
+    parent: esc-development
+    weight: 2
+---
+
+Pulumi Service Provider empowers you to manage your entire infrastructure and application landscape through a unified approach. This means you can define [environments](/docs/esc/environments/), add [version tags](/docs/esc/environments/#tagging-versions), and even control access using familiar Infrastructure as Code (IaC) practices ensuring consistency and repeatability across your deployments.
+
+Here's a list of ESC resources available through Pulumi Service Provider:
+
+- **Environment**: Define new environments and create new revisions using the normal `pulumi up` command. Learn more [here](/registry/packages/pulumiservice/api-docs/environment/).
+- **Environment Team Permission**: Control permissions to your environments. Combines with **Team** and **Environment** resources easily! Learn more [here](/registry/packages/pulumiservice/api-docs/teamenvironmentpermission/).
+- **Environment Revision Tag**: Add tags to your environment revision for flexible environment import and version control. Learn more [here](/registry/packages/pulumiservice/api-docs/environmentversiontag/).
+- **ESC Webhook**: Create webhooks to monitor or react to events on your environments, at organization level or a specific environment only. Learn more [here](/docs/esc/webhooks).
```

Comment:
How about `Create webhooks to monitor or react to events on your environments. This can be configured for specific environments or at the organization level.`